### PR TITLE
Order should save current user email

### LINKED ISSUE #604: User's e-mail should be copied to Order model
Now we're saving only anonymous user email in order model, we should always copy the e-mail address.

### PR DESCRIPTION
**Pull request by maciekjablonski**
Fixes #604

### CODE REVIEW

File: saleor/order/models.py [hunk 1]
```diff
@@ -142,7 +137,7 @@ def get_delivery_total(self):
                    Price(0, currency=settings.DEFAULT_CURRENCY))
 
     def send_confirmation_email(self):
-        email = self.get_user_email()
+        email = self.user_email
```
Comment (patrys) [inline saleor/order/models.py]: Not sure about this change. There is no way to change this email for logged in users.

File: tests/test_checkout_integration.py [hunk 2]
```diff
@@ -186,3 +189,53 @@ def test_client_login(request_cart_with_item, client, admin_user):
...
+
+    #get token:
+    token = EmailChangeRequest.objects.get(email='new@example.com').token
+    url = reverse('registration:change_email', args=[token])
+    authorized_client.get(url)
+
+    updated_user = User.objects.get(pk=normal_user.pk)
+    order = Order.objects.get(pk=order.pk)
+    assert updated_user.email != normal_user.email
+    assert updated_user.email == 'new@example.com'
```
Comment (patrys) [inline tests/test_checkout_integration.py]: This line and the line above check the same thing, don't they?

File: tests/test_checkout_integration.py [hunk 3]
```diff
@@ -186,3 +189,53 @@ def test_client_login(request_cart_with_item, client, admin_user):
     assert get_redirect_location(response) == '/'
     response = client.get(reverse('checkout:shipping-address'))
     assert response.context['checkout'].cart.token == request_cart_with_item.token
+
+
+def test_email_is_saved_in_order(authorized_client, billing_address, normal_user,  # pylint: disable=R0914
```
Comment (patrys) [inline tests/test_checkout_integration.py]: Can we rename `normal_user` to `customer_user`?

**Comment by maciekjablonski** [comment]
@patrys i cleaned code

**Pull request merged**